File: Facebook.NET/GraphAPI.Currency.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        [DataContract]
        public sealed class Currency
        {
            [DataMember(Name = "user_currency")]
            public string UserCurrency { get; set; }

            [DataMember(Name = "currency_exchange")]
            public double? CurrencyExchange { get; set; }

            [DataMember(Name = "currency_exchange_inverse")]
            public double? CurrencyExchangeInverse { get; set; }

            [DataMember(Name = "usd_exchange")]
            public double? USDExchange { get; set; }

            [DataMember(Name = "usd_exchange_inverse")]
            public double? USDExchangeInverse { get; set; }

            [DataMember(Name = "currency_offset")]
            public double? CurrencyOffset { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Currency.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Currency
            <DataMember(Name:="user_currency")>
            Public Property UserCurrency() As String
            <DataMember(Name:="currency_exchange")>
            Public Property CurrencyExchange() As System.Nullable(Of Double)
            <DataMember(Name:="currency_exchange_inverse")>
            Public Property CurrencyExchangeInverse() As System.Nullable(Of Double)
            <DataMember(Name:="usd_exchange")>
            Public Property UsdExchange() As System.Nullable(Of Double)
            <DataMember(Name:="usd_exchange_inverse")>
            Public Property UsdExchangeInverse() As System.Nullable(Of Double)
            <DataMember(Name:="currency_offset")>
            Public Property CurrencyOffset() As System.Nullable(Of Double)
        End Class
    End Class
End Namespace
